### PR TITLE
docs: sd-paragraph docs [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

**Comment by coraliefeil** [comment]
Pls fix error @auroraVasconcelos thx

**Comment by yoezlem** [comment]
@auroraVasconcelos deployment failed. 
So Cannot review it

FYI: @smfonseca can you help here?

**Comment by auroraVasconcelos** [comment]
Deployment fixed
FYI: @coraliefeil and @yoezlem

File: packages/components/src/styles/paragraph/paragraph.test.stories.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  render: (args: any) => {
+    return generateTemplate({
+      options: { templateContent: '<p class="%CLASSES%">%SLOT%</p>' },
+      args
+    });
+  }
+};
+
+/**
+ * Use the `inverted` class to make a paragraph with inverted colors.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/paragraph/paragraph.test.stories.ts:52]:
```suggestion
 * Use the `inverted` class when displayed on primary background.
```

File: packages/components/src/styles/paragraph/paragraph.stories.ts [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
 
 /**
- * A paragraph is used to display blocks of text. It uses the base font size and can contain bold and/or link styles.<br>
- * <br>
- * <b>Sizes</b>
- * <li>lg is the default paragraph size.</li>
- * <li>sm can be used as an alternative for tighter spaces.</li>
+ * Used to display blocks of text.
+ * It uses the base font size and can contain bold and/or link styles.
  */
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/paragraph/paragraph.stories.ts:12]: For the 1st sample can you make it look like FIGMA?

"Large Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique. Facilisis commodo integer hendrerit tortor."

File: packages/components/src/styles/paragraph/paragraph.stories.ts [hunk 3]
```diff
@@ -41,22 +39,41 @@ export const Default = {
   }
 };
 
+/**
+ * Use the `sd-paragraph--size-` class to set the size:
+ * - `sd-paragraph--size-default`
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/paragraph/paragraph.stories.ts:44]: We need to create a ticket to fix the name of this variant as in FIGMA the default is named as Large

**Review by smfonseca — APPROVED** [review]

File: packages/components/src/styles/paragraph/paragraph.stories.ts (hunk 3 above)
Comment (auroraVasconcelos) [reply packages/components/src/styles/paragraph/paragraph.stories.ts:44]: There was a mistake, default doesn't exists. `sd-paragraph--size-default` will delete.

File: packages/components/src/styles/paragraph/paragraph.stories.ts (hunk 2 above)
Comment (auroraVasconcelos) [reply packages/components/src/styles/paragraph/paragraph.stories.ts:12]: Done

File: packages/components/src/styles/paragraph/paragraph.stories.ts (hunk 3 above)
Comment (auroraVasconcelos) [reply packages/components/src/styles/paragraph/paragraph.stories.ts]: Done

File: packages/components/src/styles/paragraph/paragraph.stories.ts [hunk 4]
```diff
@@ -41,22 +44,39 @@ export const Default = {
   }
 };
 
+/**
+ * Use the `sd-paragraph--size-sm` class to set a smaller font size.
+ */
+
+export const Size = {
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/paragraph/paragraph.stories.ts:51]: Can we add this?
"
Large is the default paragraph size
sd-paragraph--size-sm: Small can be used as an alternative"

File: packages/components/src/styles/paragraph/paragraph.stories.ts [hunk 5]
```diff
@@ -41,22 +44,39 @@ export const Default = {
...
+        Large Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+        phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
+        Facilisis commodo integer hendrerit tortor.
+      </p>
+    </div>
+  `
+};
+
 /**
  * Use the `inverted` class to make a paragraph with inverted colors.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/paragraph/paragraph.stories.ts:69]:
```suggestion
 * Use the `inverted` class when displayed on primary background.
```

Comment (auroraVasconcelos) [reply packages/components/src/styles/paragraph/paragraph.stories.ts]: Updated

File: packages/components/src/styles/paragraph/paragraph.stories.ts (hunk 4 above)
Comment (auroraVasconcelos) [reply packages/components/src/styles/paragraph/paragraph.stories.ts:50]: Updated

**Review by MartaPintoTeixeira — APPROVED** [review]

**Pull request merged**